src/Microsoft.VisualStudio.ProjectSystem.Managed.VS/ProjectSystem/VS/Properties/VisualBasic/SplashScreenEnumProvider.cs

```diff
@@ -0,0 +1,98 @@
+﻿// Licensed to the .NET Foundation under one or more agreements. The .NET Foundation licenses this file to you under the MIT license. See the LICENSE.md file in the project root for more information.
+
+using Microsoft.Build.Framework.XamlTypes;
+using Microsoft.CodeAnalysis;
+using Microsoft.VisualStudio.LanguageServices;
+using Microsoft.VisualStudio.LanguageServices.Implementation.ProjectSystem;
+using Microsoft.VisualStudio.ProjectSystem.Properties;
+
+namespace Microsoft.VisualStudio.ProjectSystem.VS.Properties.VisualBasic;
+
+[ExportDynamicEnumValuesProvider("SplashScreenEnumProvider")]
+[AppliesTo(ProjectCapability.VisualBasic)]
+internal class SplashScreenEnumProvider : IDynamicEnumValuesProvider
+{
+    private readonly Workspace _workspace;
+    private readonly UnconfiguredProject _unconfiguredProject;
+
+    public SplashScreenEnumProvider([Import(typeof(VisualStudioWorkspace))] Workspace workspace, UnconfiguredProject unconfiguredProject)
```

Comment:
For this MEF part to work, you'll need `[ImportingConstructor]` here.

Reply:
good catch!